soh/soh/Enhancements/randomizer/3drando/item_location.cpp

```diff
@@ -777,6 +777,43 @@ void LocationTable_Init() {
+    locationTable[ZR_OPEN_GROTTO_BEEHIVE_RIGHT]                  = ItemLocation::Base   (RC_ZR_OPEN_GROTTO_BEEHIVE_RIGHT,           0, "ZR Open Grotto Beehive Right",           ZR_OPEN_GROTTO_BEEHIVE_RIGHT,           RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_ZORAS_RIVER);
+    locationTable[ZR_STORMS_GROTTO_BEEHIVE]                      = ItemLocation::Base   (RC_ZR_STORMS_GROTTO_BEEHIVE,               0, "ZR Storms Grotto Beehive",               ZR_STORMS_GROTTO_BEEHIVE,               RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_ZORAS_RIVER);
+    locationTable[ZD_IN_FRONT_OF_KING_ZORA_BEEHIVE_LEFT]         = ItemLocation::Base   (RC_ZD_IN_FRONT_OF_KING_ZORA_BEEHIVE_LEFT,  0, "ZD In Front of King Zora Beehive Left",  ZD_IN_FRONT_OF_KING_ZORA_BEEHIVE_LEFT,  BLUE_RUPEE, {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_ZORAS_DOMAIN);
+    locationTable[ZD_IN_FRONT_OF_KING_ZORA_BEEHIVE_RIGHT]        = ItemLocation::Base   (RC_ZD_IN_FRONT_OF_KING_ZORA_BEEHIVE_RIGHT, 0, "ZD In Front of King Zora Beehive Right", ZD_IN_FRONT_OF_KING_ZORA_BEEHIVE_RIGHT, RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_ZORAS_DOMAIN);
+    locationTable[ZD_BEHIND_KING_ZORA_BEEHIVE]                   = ItemLocation::Base   (RC_ZD_BEHIND_KING_ZORA_BEEHIVE,            0, "ZD Behind King Zora Beehive",            ZD_BEHIND_KING_ZORA_BEEHIVE,            RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_ZORAS_DOMAIN);
+    locationTable[LH_GROTTO_BEEHIVE]                             = ItemLocation::Base   (RC_LH_GROTTO_BEEHIVE,                      0, "LH Grotto Beehive",                      LH_GROTTO_BEEHIVE,                      RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_LAKE_HYLIA);
+    locationTable[GV_DEKU_SCRUB_GROTTO_BEEHIVE]                  = ItemLocation::Base   (RC_GV_DEKU_SCRUB_GROTTO_BEEHIVE,           0, "GV Deku Scrub Grotto Beehive",           GV_DEKU_SCRUB_GROTTO_BEEHIVE,           RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_GERUDO_VALLEY);
+    locationTable[COLOSSUS_GROTTO_BEEHIVE]                       = ItemLocation::Base   (RC_COLOSSUS_GROTTO_BEEHIVE,                0, "Colossus Grotto Beehive",                COLOSSUS_GROTTO_BEEHIVE,                RED_RUPEE,  {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_KAKARIKO);
```

Comment:
Colossus grotto is in  SpoilerCollectionCheckGroup::GROUP_KAKARIKO for some reason